api/src/main/resources/liquibase-update-to-2.3.xml

```diff
@@ -33,4 +33,16 @@
     	<renameColumn tableName="allergy" oldColumnName="comment" newColumnName="comments" columnDataType="varchar(1024)"/>
     </changeSet>
 
+    <changeSet id="TRUNK-5413" author="odorajonathan" dbms="mysql">
+      <preConditions onFail="MARK_RAN">
+            <not><columnExists tableName="concept" columnName="category"/></not>
+      </preConditions>
+      <comment>
+          Add Category field to OrderSet
+      </comment>
+      <addColumn tableName="concept">
+            <column name="category" type="varchar(50)"/>
```

Comment:
HI @dkayiwa Travis fails here on this PR but as far as other changesets with other fields of type concept are, most of them are of datatype `varchar(255)` or `varchar(50)` yet both cause fails. Some of the fields i saw were role, speciality, units and a few others whose changesets contain that. Need some guidance for this part

Reply:
Can you point me to any changeset that has a concept id column?

Reply:
I looked at some lines like this [line](https://github.com/openmrs/openmrs-core/blob/3883edf72fe0355cbdc327f36cbdd9dfd890ce06/api/src/main/resources/liquibase-update-to-2.0.xml#L3399) and [this](https://github.com/openmrs/openmrs-core/blob/3883edf72fe0355cbdc327f36cbdd9dfd890ce06/api/src/main/resources/liquibase-schema-only.xml#L317) all are of type concept

Reply:
How do you tell that those ones are concept fields?

Reply:
First of all I checked for their classes then their hibernate mappings and then the liquibase to confirm they are concepts

Reply:
Can you show me their classes where the type is concept?

Reply:
Here are the classes for [severity](https://github.com/openmrs/openmrs-core/blob/3883edf72fe0355cbdc327f36cbdd9dfd890ce06/api/src/main/java/org/openmrs/Allergy.java#L34) and [units](https://github.com/openmrs/openmrs-core/blob/3883edf72fe0355cbdc327f36cbdd9dfd890ce06/api/src/main/java/org/openmrs/DrugIngredient.java#L27)

Reply:
Are you sure you are looking at the correct classes?

Reply:
I thought those are the classes containing the concept fields

Reply:
Database table names map to specific Java classes. You can look those up in the hibernate mapping files.

Reply:
I thought those are the classes containing the concept fields

Reply:
[HERE ](https://github.com/openmrs/openmrs-core/blob/3883edf72fe0355cbdc327f36cbdd9dfd890ce06/api/src/main/resources/org/openmrs/api/db/hibernate/Allergy.hbm.xml#L35)is the Allergy.hbm.xml where severity is found and [THIS](https://github.com/openmrs/openmrs-core/blob/3883edf72fe0355cbdc327f36cbdd9dfd890ce06/api/src/main/resources/org/openmrs/api/db/hibernate/DrugIngredient.hbm.xml#L31) for the DrugInredient.hbm.xml. Both point to concept class but i don't still get it

Reply:
Is the database table name referenced in the hibernate mapping file the same as that in the liquibase changeset?

Reply:
Nope, now that I've noticed it

Reply:
@dkayiwa I failed getting same DB tablenames to get the specified classes as in the changesets, they seem to differ.

Reply:
Which exact database table names and classes are specified there?

Reply:
I mean the complete set of (liquibase changeset, hibernate mapping file, database table name, and database column name)

Reply:
This is the [class ](https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/Allergy.java)for concept field severity, it's [HBM](https://github.com/openmrs/openmrs-core/blob/3883edf72fe0355cbdc327f36cbdd9dfd890ce06/api/src/main/resources/org/openmrs/api/db/hibernate/Allergy.hbm.xml#L35) and the [liquibase ](https://github.com/openmrs/openmrs-core/blob/a346ecbcfdc2a4c14cbed1be6a054ed362713690/api/src/main/resources/liquibase-update-to-2.0.xml#L3399) been doing the same for other concepts all having the same approach

Reply:
Are those pointing to the same table?

Reply:
Been lookin for the same table names, that's been holding me down.

Reply:
Did you understand my question?

Reply:
No, they don't have the same table

Reply:
So why don't you look at liquibase changesets that have a table name matching the hibernate mapping file and java class name?